src/Stratis.Bitcoin.Features.Wallet.Tests/WalletManagerTest.cs

```diff
@@ -301,10 +298,11 @@ public void UpdateLastBlockSyncedHeightWhileWalletCreatedDoesNotThrowInvalidOper
         {
             DataFolder dataFolder = CreateDataFolder(this);
             var loggerFactory = new Mock<ILoggerFactory>();
-            loggerFactory.Setup(l => l.CreateLogger(It.IsAny<string>())).Returns(new Mock<ILogger>().Object);
+            loggerFactory.Setup(l => l.CreateLogger(It.IsAny<string>()))
+               .Returns(new Mock<ILogger>().Object);
 
-            var walletManager = new WalletManager(loggerFactory.Object, this.network, this.chain, NodeSettings.Default(), new Mock<WalletSettings>().Object,
-                                                  dataFolder, new Mock<IWalletFeePolicy>().Object, new Mock<IAsyncLoopFactory>().Object, new NodeLifetime(), DateTimeProvider.Default);
+            var walletManager = new WalletManager(loggerFactory.Object, Network.Main, new Mock<ConcurrentChain>().Object, NodeSettings.Default(), new Mock<WalletSettings>().Object,
```

Comment:
@fassadlr Should this be using Network.Main here?

Reply:
Will fix 👍

Reply:
Actually @rowandh I see now that Network.Main is used like that in that test. That file is a mess with network usage and is not consistent throughout. We can fix this after at some point.

Reply:
Ok, no worries. I only ask because I remember seeing a PR a while back to clean this up, but it does still seem to be inconsistent.